FinSearchUnified/Bundle/StoreFrontBundle/Gateway/Findologic/Hydrator/CustomListingHydrator.php

```diff
@@ -54,14 +54,14 @@ public function hydrateFacet(SimpleXMLElement $select)
      */
     public function getFormFieldName($name)
     {
-        $escapedName = preg_replace(
-            '/[^\xC3\x96|\xC3\x9C|\xC3\x9F|\xC3\xA4|\xC3\xB6|\xC3\xBC|\x00-\x7F]|[\.\s\x5B]/',
+        $escapedName = mb_ereg_replace(
```

Comment:
A comment might be helpful here, which chars are getting replaced?

Reply:
Please see the method docblock.